opera-developer/PKGBUILD

```diff
@@ -39,6 +39,6 @@ package() {
 	
 	unlink ${pkgdir}/usr/bin/${pkgname}
 	cd ${pkgdir}/usr/bin
-	ln -sr ../usr/lib/opera-developer/opera-developer
+	ln -sf /usr/lib/${pkgname}/${pkgname}
 }
```

Comment:
your symlink does not point to the actual binary

you could change it to
`ln -sr ../lib/opera-developer/opera-developer`

or make it more obvious with the
`ln -sf /usr/lib/${pkgname}/${pkgname}`